README.md

```diff
@@ -108,7 +108,12 @@ Additionally check out the sparrow [configuration](#configuration) variants.
 
 ## Usage
 
-Use `sparrow run` to execute the instance using the binary.
+Use `sparrow run` to execute the instance using the binary. A `name` (a valid DNS name) is required to be passed, else
```

Comment:
sw `name` to `sparrowName` here as well. 

Maybe add a hint that the `sparrowName` needs to be unique and valid for the sparrow instance to guarantee proper managing of global sparrow targets

Reply:
I will fix it

Reply:
thanks 👍 fixed